source/content/migrate-manual.md

```diff
@@ -146,9 +146,22 @@ Your **code** is all custom and contributed modules or plugins, themes, and libr
 
   Refer to the "Custom and contrib parts of your Drupal project" section of [Basic Directory Structure of a Drupal 7 Project](https://www.drupal.org/node/2621480) for more details.
 
-  </Tab>
+<Alert title="Note" type="info" >
+
+  If you are using Multisite and wanted to migrate a sub-site, you will need to prepare the directory and the database first:
+  1. Copy the base site to a new directory.
+  1. Delete all sites, with the exception of"
+    - `sites/{sitename}`: the site they are trying to migrate
+    - `sites/all`: contains all modules and themes.
+  1. Rename `sites/{sitename}`  to `sites/default`.
+
+</Alert>
 
-  <Tab title="Drupal 8" id="d8-code">
+<Partial file="export-database.md" />
```

Comment:
@wordsmither We should also remove this line since we have an export database below the docs.

Reply:
@v0jpantheon  - not sure what you're referring to when you say "below in the docs"...

Reply:
since we have a section already, we don't need database instructions there: https://pr-7355--pantheon-docs.my.pantheonfrontend.website/docs/migrate-manual#add-your-database